pkg/httputil/logging.go

```diff
@@ -53,13 +54,27 @@ func RequestID(r *http.Request) (*http.Request, string) {
 	return r, reqID
 }
 
+func SourceIP() string {
```

Comment:
This function will perform a DNS lookup per log line. There are multiple reasons not to do this: it's slow, leaks information, can fail, won't yield information for internal IPs, loss information for external IPs, needs to be configured to use a different server from some networks, etc.

Could we just not do this, and instead use the text format of the IP address? It will look like "10.20.30.40", which is now than adequate.